wasm/build.js

```diff
@@ -19,21 +20,28 @@ async function buildRollup(input, output) {
 async function buildWasm(network) {
     await buildRollup({
         input: {
-            "aleo_wasm": "entry",
+            "aleo_wasm": "./Cargo.toml",
+            "aleo_wasm_custom": "./Cargo.toml?custom",
         },
         plugins: [
-            virtual({
-                "entry": `export { default } from "./Cargo.toml";`,
-            }),
-
             rust({
-                cargoArgs: [
-                    // This enables multi-threading
-                    "--config", `build.rustflags=["-C", "target-feature=+atomics,+bulk-memory,+mutable-globals", "-C", "link-arg=--max-memory=4294967296"]`,
-                    "--no-default-features",
-                    "--features", `browser,${network}`,
-                    "-Z", "build-std=panic_abort,std",
-                ],
+                extraArgs: {
+                    cargo: [
+                        "--config", "build.rustflags=" + JSON.stringify([
+                            // This enables multi-threading
+                            "-C", "target-feature=+atomics,+bulk-memory,+mutable-globals",
+                            "-C", "link-arg=--max-memory=4294967296",
+
+                            // Strips out debug information
+                            "-Z", "location-detail=none",
+                            "-Z", "fmt-debug=none",
```

Comment:
Does this takeout any console.log statements?

Reply:
No, it just removes all uses of `{:?}`.